RealtimeCSG/Assets/Plugins/RealtimeCSG/Editor/Scripts/Control/Managers/SceneViewEventHandler.cs

```diff
@@ -70,8 +70,8 @@ internal static void OnScene(SceneView sceneView)
 				TooltipUtility.InitToolTip(sceneView);
 				if (Event.current.type == EventType.Repaint)
 				{
-					SceneViewBottomBarGUI.ShowGUI(sceneView);
-                	SceneViewInfoGUI.DrawInfoGUI( sceneView );
+					//SceneViewBottomBarGUI.ShowGUI(sceneView);
+                	//SceneViewInfoGUI.DrawInfoGUI( sceneView );
```

Comment:
I vaguely remember this was a Unity bug workaround .. so might need an ifdef around it (if it's been fixed in unity)

Reply:
do you know what version this may have been introduced in?

Reply:
I'm guessing the version that introduced the new sceneview UI stuff might've changed things, so might be better to keep things the way they were for earlier unity versions, and switch to the new things for versions after

Reply:
yeah. i haven't noticed any issues cropping up from this. keeping those two lines actually causes the toolbar to be drawn twice. once in the correct place, and again above it.